app-admin/lnav/lnav-0.11.0.ebuild

```diff
@@ -12,32 +13,43 @@ SRC_URI="https://github.com/tstack/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}
 LICENSE="BSD-2"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="unicode"
+IUSE="pcap test"
+RESTRICT="!test? ( test )"
 
 RDEPEND="
 	app-arch/bzip2:0=
 	app-arch/libarchive:=
 	>=dev-db/sqlite-3.9.0
 	dev-libs/libpcre[cxx]
+	pcap? ( net-analyzer/wireshark[tshark] )
```

Comment:
Put this at the end as it's conditional? (Usually we do all unconditional sorted, then conditional sorted by USE flag name)